ca_bringup/launch/minimal_ros_control.launch

```diff
@@ -0,0 +1,20 @@
+<launch>
+  <arg name="robot_id"  default="$(optenv ID 1)"      doc="Unique identifier of the robot [1-Inf.)"/>
+  <arg name="ns" value="create$(arg robot_id)"/>
+
+  <!-- Robot description -->
+  <include file="$(find ca_description)/launch/create_description.launch" ns="$(arg ns)">
+    <arg name="robot_id" value="$(arg robot_id)"/>
+  </include>
+
+  <!-- Load robot's hw interface -->
+  <node name="robot_hw_interface" pkg="create_control" type="create_control_node" ns="$(arg ns)"
+   output="screen">
+    <rosparam command="load" file="$(find ca_driver)/config/default.yaml"/>
+  </node>
+
+  <!-- Ros control -->
+  <include file="$(find create_control)/launch/create_control.launch" ns="$(arg ns)"/>
```

Comment:
Taking into account that `ROS_CONTROL` is being used for the simulation, it won't be a bad idea to do the same in `minimal.launch`.

Reply:
Changed to use ROS_CONTROL roslaunch ca_bringup minimal.launch